diff --git a/Formula/emscripten.rb b/Formula/emscripten.rb
index 3dd8cc90b19..3a3d5f0a632 100644
--- a/Formula/emscripten.rb
+++ b/Formula/emscripten.rb
@@ -27,7 +27,7 @@ class Emscripten < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "370f65a033fb6ffa2b46dff62435c84e1436a8fadb1002cd87e4a449803084f9"
   end
 
-  depends_on "cmake" => :build
+  #depends_on "cmake" => :build
   depends_on "node"
   depends_on "python@3.11"
   depends_on "yuicompressor"
diff --git a/Formula/rust.rb b/Formula/rust.rb
index 762966b78a1..cab8797e916 100644
--- a/Formula/rust.rb
+++ b/Formula/rust.rb
@@ -33,7 +33,7 @@ class Rust < Formula
     end
   end
 
-  depends_on "cmake" => :build
+  #depends_on "cmake" => :build
   depends_on "ninja" => :build
   depends_on "python@3.11" => :build
   depends_on "openssl@3"
